readme.md

```diff
@@ -124,5 +133,5 @@ Your Answer:
 Describe some of the common git workflows for teams (fork and pull request, etc).
 
 ```text
-
+Forking gives multiple developers a server-side repository to work on. Pull requests provide an interface to evaluate proposed changes before integrating. Cloning creates an exact but local replica of a given repository on a user's machine. Branches are individual development flows that can later be merged together. They are useful if multiple developers are working on a single project, or if a user wants multiple branches to keep a complex project clean.
```

Comment:
:+1: Great! We've also named a few workflows in [this lesson](https://github.com/ga-dc/curriculum/tree/master/08-single-page-web-apps/git-teams)